src/styles/index.scss

```diff
@@ -1,3 +1,99 @@
+$max-sec: 60s;
+$max-min: $max-sec * 60;
+$full-circle: 360deg;
+$speed-up-sec: 10s;
+$speed-up-min: $speed-up-sec * 60;
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border: dotted 1vmin #000;
+  border-radius: 50%;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+
+  &::before {
+    content: '';
+    position: absolute;
+    width: 5vmin;
+    height: 5vmin;
+    border-radius: 5vmin;
+    background-color: #f6a603;
+    z-index: 2;
+  }
+
+  &__hand {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    height: 75vmin;
+  }
+
+  &__minutes-hand {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    position: relative;
+
+    animation-name: clock;
+    animation-timing-function: steps(60);
+    animation-duration: $max-min;
+    animation-fill-mode: backwards;
+    animation-iteration-count: infinite;
+    transform-origin: bottom;
+  }
+
+  &__seconds-hand {
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+    position: relative;
+    transform: translateX(5px);
+
+    animation-name: clock-sec;
+    animation-duration: $max-sec;
+    animation-timing-function: linear;
+    animation-fill-mode: backwards;
+    animation-iteration-count: infinite;
+    transform-origin: top;
+  }
+}
+
+@keyframes clock {
+  from {
+    transform: rotate(0);
+  }
+  to {
+    transform: rotate($full-circle);
+  }
+}
+
+@keyframes clock-sec {
+  from {
+    transform: rotate(0);
+  }
+  to {
+    transform: rotate($full-circle);
+  }
+}
+
+.speed-up-min {
+  animation-name: clock;
+  animation-duration: $speed-up-min;
+}
+
+.speed-up-sec {
+  animation-name: clock-sec;
+  animation-duration: $speed-up-sec;
```

Comment:
The BEM modifier classes `speed-up-min` and `speed-up-sec` should be used for testing purposes only. They should be removable without affecting the basic functionality of the stopwatch, as per the task requirements.